support/psx/psx.cpp

```diff
@@ -107,6 +123,35 @@ static void unload_cue(toc_t *table)
 	memset(table, 0, sizeof(toc_t));
 }
 
+static int load_chd(const char *filename, toc_t *table)
+{
+
+  unload_chd(table);
+  chd_error err = mister_load_chd(filename, table);
+  if (err != CHDERR_NONE)
+  {
+    return 0;
+  }
+
+  for(int i = 0; i < table->last; i++)
```

Comment:
Does the compiler optimize out the i++ to make it ++i in machine code?

Reply:
table->last is not the last after processing, it's total number of tracks. If you are talking about it.